tasks/main.yml

```diff
@@ -34,11 +37,19 @@
     mode: 0664
   notify: Restart Cerebro
 
+- name: Fix Cerbero ownership
+  file:
+    state: directory
+    dest: "{{ cerebro_install_dir }}"
+    owner: "{{ cerebro_owner }}"
+    group: "{{ cerebro_group }}"
+    mode: 0755
+
```

Comment:
I think  we would have to restart the service if the directory ownership changes might need another notify here.